controllers/form-instances.ts

```diff
@@ -113,4 +120,51 @@ formInstanceRouter.delete(
   },
 );
 
+formInstanceRouter.post(
+  '/custom-form-type/create',
+  async (req: Request, res: Response) => {
+    if (!req.body.name) {
+      throw new HttpError('No name was provided.', 400);
+    }
+
+    const formType = await createCustomFormType(req.body.name);
+    res.status(201).send({
+      id: formType.id,
+    });
+  },
+);
+
+formInstanceRouter.get(
+  '/custom-form-type/list',
```

Comment:
don't think you need this and its create counterpart. the forms are enough. and you should be able to fetch them through resources